### PR TITLE
Add function for calculating errors using fisher information

### PR DESCRIPTION
**Pull request by ibab**

### CODE REVIEW

**Comment by chrisburr** [comment]
Is this ready for review now?

**Comment by ibab** [comment]
Yup

File: tests/test_stats.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    model.observed(X)
+    model.initialize({
+        mu: 2,
+        sigma: 2,
+    })
+    np.random.seed(0)
+    xs = np.random.normal(0, 1, 200)
+    model.fit(xs)
+    cov = fisher(model)
+    assert np.isclose(cov[mu][mu], 0.00521652579559)
```
Comment (chrisburr) [inline tests/test_stats.py]: Add a comment explaining `0.00521652579559`?

**Comment by chrisburr** [comment]
Finally, our longest standing pull request can be closed :wink: :+1:

**Pull request merged**